### PR TITLE
Improve y-axis logic for unit, scalingFactor, mDenom.

### PR DESCRIPTION
**Pull request by tjann**

### CODE REVIEW

File: static/js/plot.js [hunk 1]
```diff
@@ -233,15 +233,39 @@ function drawTimeSeries(seriesArray, valueKey, selector) {
...
+  // further fine-tuning.
   let yLabelText = "";
   if (sample["unit"]) {
     yLabelText = sample["unit"];
   }
-  if (sample["scalingFactor"] == 100) {
-    yLabelText = "% of " + sample["measurementDenominator"];
-  } else if (sample["scalingFactor"]) {
-    yLabelText = "Proportion of " + sample["measurementDenominator"];
+  if (sample["measurementDenominator"]) {
```
Comment (shifucun) [inline static/js/plot.js:244]: would be nice have a comment of some sample input and what the label should be? Later can move them to unittest

Comment (tjann) [reply static/js/plot.js:258]: Done

**Comment by tjann** [comment]
Motivating case:

Local

Current prod


Other changes:
Local 1
Current prod 1

Local 2
Current prod (BUG)



Unchanged:
Unit only--local

Unit only--current prod

Debt percent--local

Debt percent--current prod

**Comment by tjann** [comment]
For future PR: add hyperlink to the StatVars in yLabel.

**Pull request merged**